test/test_timeout.rb

```diff
@@ -159,4 +159,24 @@ def test_fork
     assert_equal 'timeout', r.read
     r.close
   end
+
+  def test_threadgroup
+    omit 'fork not supported' unless Process.respond_to?(:fork)
```

Comment:
`assert_separately` is better so it's also tested on non-fork platforms, and the subprocess can use `assert_equal` etc

Reply:
`fork` is much faster than `assert_separately`, so I thought to not slow down the tests for some corner case. Nevertheless I changed it.